threejs/three.d.ts

```diff
@@ -5791,6 +5792,8 @@ declare module THREE {
     export class WireframeHelper extends Line {
         constructor(object: Object3D, hex?: number);
 
+        object: Object3D;

```

Comment:
WireframeHelper class does not seem to have the 'object' property.

https://github.com/mrdoob/three.js/blob/master/src/extras/helpers/WireframeHelper.js